app/controllers/todos_controller.rb

```diff
@@ -23,11 +23,26 @@ def create
     render json: '', status: :unprocessable_entity
   end
 
+  def update
+    todo = Todo.find(params[:id])
+    todo.update!(update_todo_params)
```

Comment:
We should be able to chain those commands.

Reply:
I could do this in one line, but I think it is uglier. `todo = Todo.find(params[:id]).tap { _1.update!(update_todo_params) }`.